test/openssl/test_ssl.rb

```diff
@@ -206,6 +206,42 @@ def test_getbyte
     }
   end
 
+  def test_gets_chomp
```

Comment:
I think this is better fit in `test/openssl/test_pair.rb`, where we have test code for most of OpenSSL::Buffering methods.

[`test_getbyte` test case probably should be moved too, but this refactoring would be out of scope of this PR.]